docs/index.rst

```diff
@@ -24,6 +24,7 @@ NetsBlox Documentation
     adv/deploy.rst
     adv/cli.rst
     adv/rpc-support.rst
+    adv/configure-alexa-service.rst
```

Comment:
I guess we didn't want this info in the Alexa service docs since that's more student-oriented. But we could put it in `rpc-support.rst` somewhere, maybe just after the table of basic info for api keys.